atomics/T1053.005/T1053.005.yaml

```diff
@@ -0,0 +1,19 @@
+attack_technique: T1053.005
+display_name: "Scheduled Task/Job: Scheduled Task"
+atomic_tests:
+- name: Scheduled Task Startup Script
+  auto_generated_guid: fec27f65-db86-4c2d-b66c-61945aee87c2
+  description: |
+    Run an exe on user logon or system startup.  Upon execution, success messages will be displayed for the two scheduled tasks. To view
+    the tasks, open the Task Scheduler and look in the Active Tasks pane.
+  supported_platforms:
+  - windows
+  executor:
+    command: |
+      schtasks /create /tn "T1053_005_OnLogon" /sc onlogon /tr "cmd.exe /c calc.exe"
+      schtasks /create /tn "T1053_005_OnStartup" /sc onstart /ru system /tr "cmd.exe /c calc.exe"
+    cleanup_command: |
+      schtasks /delete /tn "T1053_005_OnLogon" /f >nul 2>&1
+      schtasks /delete /tn "T1053_005_OnStartup" /f >nul 2>&1
+    name: command_prompt
+    elevation_required: true
```

Comment:
Since this yaml file already exists in @MHaggis PR which has already been merged, we need to just take this atomic and add it to the existing file and remove it from this PR.